lib/DDG/Spice/IslamicPrayerTimes.pm

```diff
@@ -0,0 +1,52 @@
+package DDG::Spice::IslamicPrayerTimes;
+# ABSTRACT: Shows Islamic prayer times according to given or user's current location and time
+
+use strict;
+use DDG::Spice;
+use Text::Trim;
+
+spice is_cached => 1;
+
+name "IslamicPrayerTimes";
+description "Islamic prayer times for the current day and given location";
+primary_example_queries "islamic prayer times", "fethiye namaz times", "salat times in dc", "namaz times for san marino";
+category "time_sensitive";
+topics "special_interest", "everyday";
+
+code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/IslamicPrayerTimes.pm";
+attribution github  => ["https://github.com/ozdemirburak", "Burak Özdemir"],
+            twitter => ["https://twitter.com/ozdemirbur", "Burak Özdemir"],
+            web     => ["http://burakozdemir.co.uk", "Burak Özdemir"];
+
+spice to => 'http://muslimsalat.com/$1/weekly.json?key={{ENV{DDG_SPICE_MUSLIMSALAT_APIKEY}}}';
+spice wrap_jsonp_callback => 1;
```

Comment:
Looks like the API supports a `callback` param? Let's add `&callback={{callback}}` to `spice to`